packages/graphql/index.ts

```diff
@@ -0,0 +1,2 @@
+// This is an empty typescript file (which lives next to tsconfig.json) to satisfy my IDE.
```

Comment:
Did you try this solution? 
https://stackoverflow.com/a/60234067

Reply:
Yes I tried, but it doesn't matter, because in `src/` there aren't any TS files. I think the empty file is nice temporary solution until we convert to TS. Or we can delete the tsconfig file... 

Let's discuss later